stock_release_channel_shipment_advice_process_end_time/models/res_company.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+
+    _inherit = "res.company"
+
+    release_channel_shipment_advice_arrival_delay = fields.Integer(
+        help="Global default value for the delay between the release channel process "
+        "end time and the arrival of shipments to the dock."
+    )
```

Comment:
I find this overkill. Having it configured on the warehouse is enough.

The stock_release_channel should be first made multi-company aware. It is useless to define a company setting while the model needing this is not bound to a company. This is a puzzle piece missing

Reply:
Nevertheless, ideally also add in another PR the missing company field on stock release channel + multi-company rules.
Even if not used here. This was just missing in the past.